### PR TITLE
#9 Skeleton SVG Framework

### LINKED ISSUE #9: Create SVG Viewport
Requirements:
 - Embedded SVG fills page
 - Skeleton framework for app

### PR DESCRIPTION
**Pull request by Zaskoda**
Closes #9

### CODE REVIEW

File: client/src/App.vue [hunk 1]
```diff
@@ -19,27 +26,51 @@ export default {
...
+    },
+    openMenu() {
+      this.menu = !this.menu
+    },
+    roll() {
+      alert('you rolled the dice')
     }
   }
 }
 </script>
```
Comment (excalq) [inline client/src/App.vue:39]: Would it be better for maintainability to have the template imported from a view file? (Or whatever the Vue equivalent is)

File: client/src/components/layouts/SvgContainer.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<script>
```
Comment (excalq) [inline client/src/components/layouts/SvgContainer.vue:1]: It's unclear the role this SvgContainer serves. Is it for any SVG, or the dice slots themselves in the app?

**Review by excalq — APPROVED** [review]
Looks super cromulent! Let's ship :ship: :rocket:

File: client/src/components/layouts/SvgContainer.vue (hunk 2 above)
Comment (Zaskoda) [reply client/src/components/layouts/SvgContainer.vue:1]: It's to create a single embedded SVG we can use as a canvas for drawing the app...

**Pull request merged**